src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -54,23 +53,29 @@ public static void DownloadData_InvalidArguments_ThrowExceptions()
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null); });
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null, null); });
 
-            AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataTaskAsync((string)null); });
+            if (!PlatformDetection.IsFullFramework) // netfx has bug and returns wrong argument name in exception.
+            {
+                AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataTaskAsync((string)null); });
```

Comment:
I could modify this test and validate the expected (current) behavior with NETFX.  Not sure if that buys us anything because it really is a bug and not an expected (but different) behavior.